descriptors/annotators/TrainerAnnotator.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
If you take my comments on the implementation into account this file would not even be needed.